migrations/utils.js

```diff
@@ -36,6 +37,8 @@ async function generateDummyTokens(network, DummyToken) {
 
 async function configureTokenRegistry(network, accounts, TokenRegistry, DummyToken) {
     const OWNER = accounts[0];
+    const initialTokenHolders = accounts.slice(0, CONSTANTS.NUM_INITIAL_BALANCE_HOLDERS);
```

Comment:
This will fail if we are ever deploying to Kovan, because the accounts array is only one element long in our Kovan / Mainnet deployments

Reply:
Please fix.